spec/carrier_account_spec.rb

```diff
@@ -14,6 +14,16 @@
       # Remove the carrier account once we have tested it so we don't pollute the account with test accounts
       carrier_account.delete
     end
+
+    it 'sends FedexAccount to the correct endpoint' do
+      allow(EasyPost).to receive(:make_request).with(
+        :post, '/v2/carrier_accounts/register', nil, { carrier_account: { type: 'FedexAccount' } },
+      ).and_return({ 'id' => '123' })
+
+      response = described_class.create(type: 'FedexAccount')
+
+      expect(response).to be_an_instance_of(EasyPost::EasyPostObject)
```

Comment:
Shouldn't this return `EasyPost::CarrierAccount` instead?

Reply:
Actually, `EasyPost::Util.convert_to_easypost_object` converts to `EasyPost::EasyPostObject`, and this happens in other places also.

Reply:
Hmm, we should be able to get it to the more specific CarrierAccount type. I'll check this out early next week.

Reply:
Changing `EasyPost::EasyPostObject` here in the expectation line to `described_class` with the above ID change will correct this issue. Let me know if you have questions. Once that's done, we can merge this in.